src/db.ts

```diff
@@ -75,7 +65,49 @@ function addWatch<T>(cursor: Cursor<T>, key: string, watchId: string) {
   });
 }
 
-// Update storage on state change
-addWatch(transactionCursor, TRANSACTION_KEY, 'transaction-update');
-addWatch(settingsCursor, SETTINGS_KEY, 'settings-update');
-addWatch(accountCursor, ACCOUNT_KEY, 'account-update');
+/**
+ * Remove watchdog
+ *
+ * @param cursor - Database cursor
+ * @param watchId - Unique watch id
+ */
+function removeWatch<T>(cursor: Cursor<T>, watchId: string) {
+  cursor.removeWatch(watchId);
+}
+
+/**
+ * Initialized the storage with default values and merges them with
+ * local storage entries
+ *
+ * @returns cleanup callback
+ */
+export const initDb = () => {
```

Comment:
We don't have to export DB init, db is initialized once and shared between all apps, removeWatch is not necessary as you can't have same watchdog under given id .. I'm going to force createAPI to check if chain state ready, if yes we reuse the state :)